arch/sim/src/sim/up_initialize.c

```diff
@@ -281,4 +282,9 @@ void up_initialize(void)
 #if defined(CONFIG_FS_SMARTFS) && (defined(CONFIG_SIM_SPIFLASH) || defined(CONFIG_SIM_QSPIFLASH))
   up_init_smartfs();
 #endif
+
+#ifdef CONFIG_AUDIO
+  audio_register("pcm0p", sim_audio_initialize(true));
```

Comment:
Lets set these via `CONFIG_SIM_ALSA_DEVICE`

Reply:
done

Reply:
@GUIDINGLI I don't see this in the recent commits

Reply:
@xiaoxiang781216 still hard coded here or am I not understanding